docs/deploying.rst

```diff
@@ -51,6 +51,8 @@ as a setuptools distribution.
 Hot-deploying Toil
 ------------------
 
+.. todo:: "Hot-deploying" is a confusing name. What is better?
+
 Toil can be easily deployed to a remote host, given that both Python and Toil
 are present. The first order of business after copying your workflow to each
```

Comment:
From my understanding: if you have a remote host with python and Toil installed (like anything running the Toil Docker appliance) then you have met the requirements to hot-deploy a workflow.

Hot-deployment just installs all of your dependencies for your workflow onto a running, active host.